components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ExportUtils.java

```diff
@@ -864,6 +864,7 @@ public static void addAPIMetaInformationToArchive(String archivePath, APIDTO api
             String apiType = apiDtoToReturn.getType().toString();
             API api = APIMappingUtil.fromDTOtoAPI(apiDtoToReturn, apiDtoToReturn.getProvider());
             api.setOrganization(organization);
+            apiDtoToReturn.setOrganization(organization);
```

Comment:
This is not needed because api obj has the organization and it is passed to the other methods.